packages/client-utils/src/types/index.ts

```diff
@@ -43,7 +43,6 @@ export type RequestConfig = {
  * Environmentally independent HttpClient interface.
  */
 export interface HttpClient {
-    fetch?: any;
```

Comment:
Please rembember that we are using APIClients (w/ClientUtils) in browser as well.

Reply:
yes, the field remains, it's just initialized in a different way. I think that it doesn't have to be public that's why I removed it from the interface